vmdb/app/controllers/miq_policy_controller/rsop.rb

```diff
@@ -137,6 +132,13 @@ def rsop_show_options
 
   private
 
+  def rsop_put_objects_in_sb(objects, key)
+    @sb[:rsop][key] = {}
+    objects
+      .sort_by { |o| o.name.downcase }
```

Comment:
I'm not sure this sorting is even needed, but I left it anyway.  I guess it depends how the `@sb[:rsop]` is used.